### DISCUSSION TITLE
Files related to Monitoring Hub are not available in the current Git repository.

### DISCUSSION THREAD

**Discussion by SanchanaMohankumar** [microsoft/Microsoft-Fabric-workload-development-sample#124] [Q&A] [answered]
Hi Team, 

While configuring Monitoring Hub using the [Microsoft Development Kit documentation](https://learn.microsoft.com/en-us/fabric/workload-development-kit/monitoring-hub), we noticed that the documentation references files in the Microsoft Fabric Workload Development Sample Git code. However, the Monitoring Hub files are no longer available in the latest Git repository—they were present in the initial commits but were later removed. Is there a specific reason for their removal? 

Thank you

**Comment by SanchanaMohankumar** [comment]
Hi Team, 
Can you please let us know if there is any update on this.
Thank you

**Comment by michal-mano** [comment, marked as answer]
Hi,
Thanks for bringing this to our attention. I believe that the implementation was just changed slightly without the documentation being updated, which we will address. You should still be able to find methods related to handling jobs here:
https://github.com/microsoft/Microsoft-Fabric-workload-development-sample/blob/main/Backend/src/Controllers/JobsControllerImpl.cs
